.circleci/config.yml

```diff
@@ -26,8 +26,13 @@ jobs:
               tar xzf google-cloud-sdk-*.tar.gz
               google-cloud-sdk/install.sh -q
               google-cloud-sdk/bin/gcloud components install beta cloud-datastore-emulator -q
-              google-cloud-sdk/bin/gcloud config set project testing
             fi
+      - run:
+          name: "Install kubectl "
+          command: |
+            curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
+            chmod +x ./kubectl
+            mv ./kubectl /usr/local/bin/kubectl
```

Comment:
I'm actually surprised this will be cached and brought to the next step.

Reply:
I misunderstood how CircleCI works. All the `runs` happen in the same container, not multiple ones.

Reply:
Yes. It looks like so and this makes sense. But what I don't understand is that I need to do the port forwarding in the same step as running the test.

Reply:
Probably. You can `nohup kubectl ... &` if you want to try.

Reply:
it doesn't work :(